packages/web-fragments/src/elements/fragment-host.ts

```diff
@@ -109,9 +109,12 @@ export class FragmentHost extends HTMLElement {
 
 					// CSSStyleSheet.insertRule() prepends CSS rules to the top of the stylesheet by default.
 					// We need to set the index to sheet.cssRules.length in order to append the rule and maintain specificity.
-					[...sheet.cssRules].forEach((rule) =>
-						clone.insertRule(rule.cssText, clone.cssRules.length)
-					);
+					[...sheet.cssRules].forEach((rule) => {
+						// @import directives are not allowed in Constructed Stylesheets
+						if (!(rule instanceof CSSImportRule)) {
```

Comment:
I understand that this skips over the problematic imports, but wouldn't this mean that the teleported stylesheet is different then the original stylesheet as the imports will now be missing?

In other words, why is it safe to simply drop these imports?

Reply:
This is a great context. thanks for explaining. Could you please add a shorter version as an inline comment where the import is dropped and maybe even link to this comment and atomic move spec? thanks!